pkg/apis/core/validation/validation_test.go

```diff
@@ -8089,9 +8119,105 @@ func TestValidatePodUpdate(t *testing.T) {
 					},
 				},
 			},
-			"spec: Forbidden: pod updates may not change fields",
+			"",
```

Comment:
we can have this test with this change, and add another test without varying image, "cpu only change"

Reply:
there's already a test for image change. changed this to cpu only with the same image.